lib/apex/format.ex

```diff
@@ -116,13 +116,30 @@ defimpl Apex.Format, for: Map do
   end
 
   def format(data, options) do
-    Apex.Format.Seq.format(
-      Map.to_list(data),
-      options,
-      start_token: "\%{",
-      end_token: "}",
-      numbers: false) |> colorize(data, options)
-  end
+    ap_options = :apex
+    |> Application.get_all_env
+    |> Keyword.merge(options)
+
+    data
+    |> keys_to_atoms(ap_options[:keys_to_atoms])
+    |> Map.to_list
+    |> Apex.Format.Seq.format(
+        ap_options,
+        start_token: "\%{",
+        end_token: "}",
+        numbers: false)
+    |> colorize(data, ap_options)
+  end
+
+  defp keys_to_atoms(data, shouldConvert)
+  defp keys_to_atoms(data, true) do
+    Map.new(data, fn {k,v} -> { convert_to_atom(k), v} end)
```

Comment:
There's inconsistent whitespace usage here before `convert_to_atom`:

`fn {k,v} -> { convert_to_atom(k), v} end`

Please remove the whitespace for the sake of consistency